src/VisualStudio/VisualBasic/Impl/Snippets/SnippetCompletionProvider.vb

```diff
@@ -52,6 +54,11 @@ Namespace Microsoft.VisualStudio.LanguageServices.VisualBasic.Snippets
             Dim syntaxFacts = document.GetLanguageService(Of ISyntaxFactsService)()
             Dim isPossibleTupleContext = syntaxFacts.IsPossibleTupleContext(syntaxTree, position, cancellationToken)
 
+            Dim trivia = syntaxTree.FindTriviaToLeft(position, cancellationToken)
```

Comment:
I think you want this helper: http://index/?leftProject=Microsoft.CodeAnalysis.VisualBasic.Workspaces&leftSymbol=oxebcswupkyz&file=Extensions%5CSyntaxTreeExtensions.vb

Reply:
Actually, there's probably a similar method on the ISyntaxFactsService you have above.